### ISSUE TITLE
@expo/config in @expo/webpack-config has mismatch version (running expo doctor)

### ISSUE THREAD

**Issue by elenitaex5** [expo/eas-cli#1833]
### Summary

Running `expo doctor` in my project an invalid `@expo/config-plugins` dependency where found.
`Expected package @expo/config-plugins@~6.0.0
Found invalid:
  @expo/config-plugins@4.1.1`

Searching inside my _package-lock.json_ the error comes from `@expo/webpack-config` because it's using `@expo/config 6.0.20` This package is always out of date in comparison with al expo/cli.

### Managed or bare?

managed

### Environment

```
 System:
      OS: macOS 12.6.3
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.19.0 - /opt/homebrew/opt/node@16/bin/node
      Yarn: 1.22.19 - /opt/homebrew/bin/yarn
      npm: 8.19.3 - /opt/homebrew/opt/node@16/bin/npm
      Watchman: 2023.01.16.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.11.3 - /opt/homebrew/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
    IDEs:
      Android Studio: 2021.2 AI-212.5712.43.2112.8609683
      Xcode: 14.2/14C18 - /usr/bin/xcodebuild
    npmPackages:
      @expo/webpack-config: ^18.0.4 => 18.0.4 
      expo: ~48.0.15 => 48.0.16 
      react: 18.2.0 => 18.2.0 
      react-dom: 18.2.0 => 18.2.0 
      react-native: 0.71.7 => 0.71.7 
      react-native-web: ~0.18.11 => 0.18.12 
    npmGlobalPackages:
      eas-cli: 3.10.0
      expo-cli: 6.3.7
    Expo Workflow: bare
```

`Expected package @expo/config-plugins@~6.0.0
Found invalid:
  @expo/config-plugins@4.1.1`

### Reproducible demo or steps to reproduce from a blank project

There's no code possible. The error comes building detox build for iOS.

**Comment by szdziedzic** [comment]
Did your build actually fail, or was it just a warning for the `expo-doctor`?

**Comment by elenitaex5** [comment]
It was fixed in expo/webpack-config 18.1.0 so I close this issue.
